low-level/sspl_ll_d

```diff
@@ -338,6 +341,8 @@ def main(conf_reader, systemd_support):
         logger.info('thread controller queue is ready and serving the sspl \
                      role switch State: {sspl_role_state} request received earlier')
 
+        logger.debug("Sending the sspl role switch request to"
```

Comment:
what is this for. optimize statement

Reply:
Reverted. (not required as per new commit)
Done.